src/main/java/seedu/address/commons/events/model/ToDoListChangedEvent.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.commons.events.model;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.ReadOnlyToDoList;
+
+/**
+ * Indicates the ToDoList in the modelToDo has changed
+ */
+public class ToDoListChangedEvent extends BaseEvent {
+
+    public final ReadOnlyToDoList data;
+
+    public ToDoListChangedEvent(ReadOnlyToDoList data) {
+        this.data = data;
+    }
+
+    @Override
+    public String toString() {
+        return "number of events " + data.getToDoList().size();
```

Comment:
same as above, i think we should call a "thing" in the todo list as an entry or an item, rather than an event

Reply:
In fact when I wrote the first ToDoListEvent class I call it event. Now if renaming, need to cost a lot of time to check. I don't want to do change the name lah. So troublesome.

Reply:
Yeah, we can do the refactoring next week too. No rush at the moment.